raw-span-constants/src/main/proto/org/hypertrace/core/span/constants/v1/opentelemetry.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+option java_multiple_files = true;
+package org.hypertrace.core.span.constants.v1;
+
+import "org/hypertrace/core/span/constants/v1/enum_extension.proto";
+
+// This file contains constants related to open telemetry tracer.
+// Written as per
+enum OTELSpanTag {
```

Comment:
Please add these as Java enum constants and keep the usage private to this repo.

Reply:
Done